Parsers/ASimAuthentication/Parsers/ASimAuthenticationAADServicePrincipalSignInLogs.yaml

```diff
@@ -15,6 +15,7 @@ References:
 Description: |
   This ASIM parser supports normalizing Azure Active Directory Service Principal sign in logs, stored in the AADServicePrincipalSignInLogs table, to the ASIM Authentication schema.
 ParserName: ASimAuthenticationAADServicePrincipalSignInLogs
+EquivalentBuiltInParser: _ASim_Authentication_AADServicePrincipalSignInLogs
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?

Reply:
Sorted columns